src/components/home/index.css

```diff
@@ -75,77 +72,53 @@
 
 .weCare {
   color: black;
-  margin-left: 12%;
+  margin: 11% auto 3%;
 }
 
 .pinkButtonWithWhiteText {
-  color: white;
-  border: none;
+  margin: 1.5% 0;
   background-color: #ed2a7a !important;
-  border-radius: 10px;
+  border-radius: 10px !important;
   border: none !important;
-  display: block;
-  font-size: 16px;
-  padding: 12px 28px;
-  margin: 1.5% 0;
+  height: 2.3rem;
+  width: 8rem;
 }
 
 .blueButtonWithWhiteText {
-  color: white;
-  border: none;
-  background-color: rgb(59, 175, 241);
-  border-radius: 10px;
-  font-size: 16px;
-  padding: 12px 28px;
+  background-color: #6bb2ff !important;
+  border-radius: 10px !important;
+  border: none !important;
+  height: 2.3rem;
+  width: 17rem;
   margin-bottom: 11%;
 }
 
 .weCare h1 {
-  font-family: "Noto Sans";
-  font-size: 46px;
-  font-style: normal;
-  font-weight: bold;
-  font-size: 62px;
-  line-height: 73px;
-  text-align: justify;
+  text-transform: uppercase;
+  font-family: "Noto Sans", sans-serif;
+  font-size: 3.5rem;
   color: #161b50;
+  margin-bottom: 3.5%;
 }
 
 .weCare p {
-  width: 100%;
+  font-family: roboto, sans-serif;
```

Comment:
does this need to be capitalized as Roboto?

Reply:
@louisrli Done